nimare/tests/conftest.py

```diff
@@ -15,20 +15,16 @@
 
 
 @pytest.fixture(scope='session', autouse=True)
-def download_data(tmpdir_factory):
+def download_data():
     """
-    Download 21 pain studies from NeuroVault to test IBMA functions.
+    Download and save 21 pain studies from NeuroVault to test IBMA functions.
     """
-    tst_dir = tmpdir_factory.mktemp('tests')
-    out_dir = tst_dir.ensure('resources',
-                             'data',
-                             'neurovault-data',
-                             'collection-1425',
-                             dir=True)
+    home = op.expanduser("~")
+    out_dir = op.join(home, '.nimare', 'data', 'neurovault', 'collection-1425')
```

Comment:
Why in home instead of `nimare/tests/resources`?

Reply:
At 200 MB, it's possible users will want to delete it to save space at some point, and I doubt anyone will think to look in the code directory. Putting large datasets in `~/.{package}` is pretty conventional.